e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -59,7 +61,24 @@ const typeFilters = [
 ];
 
 const { ORDERS_ID } = SAMPLE_DATABASE;
-const TEST_QUESTIONS = generateQuestions("Robert's Question", 5);
+
+const NORMAL_USER_TEST_QUESTION = {
+  name: `Robert's Super Duper Reviews`,
+  query: { "source-table": ORDERS_ID, limit: 1 },
+  collection_id: null,
+};
+
+const ADMIN_TEST_QUESTION = {
+  name: `Admin Super Duper Reviews`,
+  query: { "source-table": ORDERS_ID, limit: 1 },
+  collection_id: null,
+};
+
+// Using these names in the `last_edited_by` section to reduce confusion
+const LAST_EDITED_BY_ADMIN_QUESTION = NORMAL_USER_TEST_QUESTION;
```

Comment:
I'm using the same questions since there's no point in re-copying the question yet again. I didn't want to use the same name for `last_edited_by` since I wanted to check that it was edited by another user. In practice, before renaming, this meant saying "Admin edited the `NORMAL_USER_TEST_QUESTION`" and I found that confusing for myself. So, I changed the names to hammer home the fact that the admin did edit the normal user's question